.github/workflows/TV-check-users-in-space-developer-role.yml

```diff
@@ -0,0 +1,31 @@
+name: 'check users in space developer role'
+
+on:
+  schedule: # 1700hrs Mon-Fri
+    - cron: '00 17 * * 1-5'
+
+jobs:
+  CHECK-SPACE-USER:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: DFE-Digital/github-actions/setup-cf-cli@master
+        with:
+          CF_USERNAME: ${{ secrets.CF_USERNAME_PROD }}
+          CF_PASSWORD: ${{ secrets.CF_PASSWORD_PROD }}
+          CF_SPACE_NAME: teaching-vacancies-production
+          CF_ORG_NAME: dfe
+          CF_API_URL:  https://api.london.cloud.service.gov.uk
+          INSTALL_CONDUIT: false
+
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+        with:
+          repository: DFE-Digital/bat-infrastructure
+          path: ./remote-checkout
+
+
+      - name: Run powershell script
+        run: |
+          ./remote-checkout/scripts/check-users-in-space-developer-role.ps1 "${{ env.CF_SPACE_NAME }}" "${{secrets.SLACK_WEBHOOK}}"
+        env:
```

Comment:
This can be moved at the job level and the value reused on line 15
It will avoid duplication of setting the same value twice